lib/src/view.rs

```diff
@@ -212,8 +214,12 @@ impl View {
         }
     }
 
-    /// Iterates local/remote branch `(name, remote_ref)`s of the specified
-    /// remote in lexicographical order.
+    /// Iterates over `(name, TrackingRefPair {local_ref, remote_ref})`s for
```

Comment:
nit: remove `TrackingRefPair` and just say `(name, {local_target, remote_ref})`?

I think `TrackingRefPair` is badly named, and the exact type name is available in the function signature.

Reply:
I'm thinking of doubling down on this instead and changing the next paragraph to

> Note that, despite the name of `TrackingRefPair`, this does *not*...

But I can also remove it as you asked, especially if you have plans to rename the type. Which do you prefer?

Reply:
Isn't `{local_target, remote_ref}` enough for documentation purpose?
I'll update the type if I can find a better name, but I don't have any plan.

> Note that, despite the name of TrackingRefPair, this does not...

Perhaps, this could be added to the doc of the struct if that makes sense.

Reply:
> Isn't {local_target, remote_ref} enough for documentation purpose?

I'm not sure, but I think it's good enough, yes.

> I'll update the type if I can find a better name, but I don't have any plan.

I'm thinking of renaming it to `LocalAndRemoteRef`. It could also just be `RefPair`. I made a commit that does it, an will make a PR out of it shortly, but I haven't thought about it carefully.